lifechanger/src/app/biography/biography.component.css

```diff
@@ -0,0 +1,29 @@
+body {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+.biography {
+  margin-top: 7.6em;
+  width: 710px;
+}
+.about p {
+  color: var(--dark-content);
+  font-size: 18px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 26px;
```

Comment:
line-height property value needs to be converted from px to numbers
**Here's  hint**
![image](https://github.com/TEZv/Become_PRO_with_Angular/assets/111384399/6fe92c2e-f870-4b91-8d5c-11f5e374550b)